core/arch/arm/kernel/boot.c

```diff
@@ -1410,6 +1414,10 @@ static void get_sec_mem_from_manifest(void *fdt, paddr_t *base, size_t *size)
 		EMSG("Invalid FF-A manifest at %p: error %d", fdt, ret);
 		panic();
 	}
+
+	/* save manifest address */
+	manifest_dtb = (paddr_t)fdt;
```

Comment:
We should clear this before exiting since the address is not valid after that.

Reply:
I'm a little confused, where to exit and set manifest_dtb to NULL? Help me explain it in detail. Thank you.